product_docs/docs/pgd/5/security/role-management.mdx

```diff
@@ -2,36 +2,36 @@
 
 
-## **Roles for new nodes**
+## Roles for new nodes
 
-New PGD nodes that are added using[ bdr_init_physical](https://www.enterprisedb.com/docs/pgd/latest/reference/nodes/#bdr_init_physical) will automatically replicate the roles from other nodes of the PGD cluster.
+New PGD nodes that are added using [`bdr_init_physical`](https://www.enterprisedb.com/docs/pgd/latest/reference/nodes/#bdr_init_physical) replicate the roles from other nodes of the PGD cluster.
 
-If a PGD node is joined to a PGD group manually, without using bdr_init_physical, existing roles aren't copied to the newly joined node. This is intentional behavior to ensure that access isn't accidentally granted.
+If a PGD node is joined to a PGD group manually, without using `bdr_init_physical`, existing roles aren't copied to the newly joined node. This is intentional behavior to ensure that access isn't accidentally granted.
 
-Roles, by default, have access to all databases. Automatically copying users to a newly created node would therefore open the possibility of unintentionally exposing a database that exists on the node. If the PGD cluster had a user Alice, and Alice had been automatically created on a newly joined node, by directly connecting to that node, Alice would have access to all the databases on that node. PGD therefore doesn’t copy over existing roles and, in this situation, recommends that you copy over roles manually.
+Roles, by default, have access to all databases. Automatically copying users to a newly created node therefore opens the possibility of unintentionally exposing a database that exists on the node. If the PGD cluster had a user Alice, and Alice was automatically created on a newly joined node, by directly connecting to that node, Alice would have access to all the databases on that node. PGD therefore doesn't copy over existing roles. In this situation, we recommend that you copy over roles manually.
 
 PostgreSQL allows you to dump all roles with the command:
 
 ```
 pg_dumpall --roles-only > roles.sql
 ```
 
-You can then edit the file `roles.sql` to remove unwanted users before re-executing that on the newly created node.
+You can then edit the file `roles.sql` to remove unwanted users before executing that command again on the newly created node.
 
-When joining a new node, the “No un-replicated roles” rule also applies. If an object in the to-be replicated database relies on an unreplicated role in an (unreplicated) local database, then the join will fail when it attempts to replicate that object.
+When joining a new node, the “No unreplicated roles” rule also applies. If an object in the database being replicated relies on an unreplicated role in an (unreplicated) local database, then the join fails when it attempts to replicate that object.
```

Comment:
This happens *before* replication has actually started, during the join. Hence the to-be replicated.

Reply:
```suggestion
When joining a new node, the “No unreplicated roles” rule also applies. If an object in the database to be replicated relies on an unreplicated role in an (unreplicated) local database, then the join fails when it attempts to replicate that object.
```